L44-python-rich-status.md

```diff
@@ -0,0 +1,474 @@
+New RPC Status API for Python
+----
+* Author(s): lidizheng
+* Approver: ericgribkoff
+* Status: Draft
+* Implemented in: Python
+* Last updated: 12-10-2018
+* Discussion at: https://groups.google.com/forum/#!topic/grpc-io/8Ys6ba8gpLc
+
+## Abstract
+
+The current design of status API doesn't support rich status. Also the concept of status "details" doesn't comply with the gRPC spec. So, after discussion with @gnossen and @ericgribkoff, we think we should add a new set of status APIs to correct them.
+
+
+## Background
+
+The gRPC [Spec](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md#responses) defined two trailing data to indicate the status of an RPC call `Status` and `Status-Message`. However, status code and a text message themselves are insufficient to express more complicated situation like the RPC call failed because of quota check of specific policy, or the stack trace from the crashed server, or obtain retry delay from the server (see [error details](https://github.com/googleapis/api-common-protos/blob/master/google/rpc/error_details.proto)).
+
+The one primary use case of this feature is Google Cloud API. The Google Cloud API needs it to transport rich status from the server to the client. Also, they are the owner of those proto files.
```

Comment:
Delete the `Also` sentence (why does it matter who owns "those" proto files mentioned above?)

Reply:
Removed.